_posts/2016-05-06-elasticsearch-module_3.md

```diff
@@ -105,11 +105,11 @@ curl -XPUT 'http://localhost:9200/article_index/politics/_bulk?pretty' --data-bi
 {% endhighlight %}
 
 Como en el ejemplo estamos utilizando curl, debemos utilzar el flag --data-binary. 
-Los endpoints a esta api son */_bulk*, */{index}/_bulk* y */{index}/{type}/_bulk*. En este caso, se agregarán los datos en el índice **article_index** y serán de tipo **politics**.
 
-## Realizando la primer búsqueda
 
+> Los endpoints a esta api son */_bulk*, */{index}/_bulk* y */{index}/{type}/_bulk*. En este caso, se agregarán los datos en el índice **article_index** y serán de tipo **politics**.
```

Comment:
Estos endpoints siguen siendo en *itálica*, me parece mejor que estén como `texto preformateado`, [como dije antes](https://github.com/Desarrollo-CeSPI/desarrollo-cespi.github.io/pull/14#discussion_r63234663)..